.github/stale.yml

```diff
@@ -1,15 +1,15 @@
 # Number of days of inactivity before an issue becomes stale
-daysUntilStale: 60
+daysUntilStale: 5
```

Comment:
5 is probably too aggressive to mark as stale. Suggest to set this to 30d.  @natke any thoughts?

Reply:
Yeah, 30 days sounds good to me